wowup-electron/src/assets/i18n/en.json

```diff
@@ -72,6 +72,14 @@
         "TELEMETRY_LABEL": "Telemetry",
         "TITLE": "Application"
       },
+      "ADDON_SOURCES": {
+        "TITLE": "Addon Sources",
+        "ENABLED_SOURCES": {
+          "FIELD_LABEL": "Enabled Addon Sources",
+          "DESCRIPTION": "Select which sources may be used to update and install your addons",
+          "INPUT_LABEL": "Sources"
+        }
+      },
```

Comment:
Not sure about the translation workflow. I've manually added these values here

Reply:
I'd say stub out the translations for the other languages, but make the value "TEXT_ELEMENT". I spoke with @jliddev about this yesterday and that's what he's been doing. That makes it easy to identify what's not translated.